MetaMask/metamask-extension
# feature: confirm re-design header component implementation

## **Description**

New header component for confirmation re-design.

## **Related issues**

Fixes: #21279 

## **Manual testing steps**

1. Enable `conf-redesign` in local main build
2. Go to Test DAPP
3. Submit personal signature request
4. Check new header on confirmation page

## **Screenshots/Recordings**
![Screenshot 2023-11-29 at 4 13 15 PM](https://github.com/MetaMask/metamask-extension/assets/2182307/766680f2-1ab2-48ae-8910-2ef7312d161f)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
